utils_cv/detection/model.py

```diff
@@ -101,17 +134,45 @@ def _calculate_ap(e: CocoEvaluator) -> float:
 class DetectionLearner:
     """ Detection Learner for Object Detection"""
 
-    def __init__(self, dataset: Dataset, model: nn.Module = None):
+    def __init__(
+        self,
+        dataset: Dataset,
+        model: nn.Module = None,
+        # transform parameters
```

Comment:
i think the transform, rpn, and box parameters are redundant here. 

if the user wants to define a model, i think they should define the model.nn object then pass it in:
```python
dataset = <some dataset)
model = get_pretrained_fasterrcnn(<transform/rpn/box parameters here>)
learner = DetectionLearner(dataset, model)
```

Reply:
@jiata Very good suggestion, didn't like all these repeated parameters. Just updated this PR. Let me know if you have any other comments and if not then please approve this PR.